schema/stablecoin/transfer.sql

```diff
@@ -0,0 +1,80 @@
+CREATE TABLE stablecoin.transfer (
+    "from" bytea,
+    "to" bytea,
+    coin_name text NOT NULL,
+    symbol text,
+    decimals numeric,
+    contract_address bytea,
+    amount numeric,
+    amount_raw numeric,
+    block_time timestamptz NOT NULL,
+    tx_hash bytea,
+    evt_index integer
+);
+
+CREATE OR REPLACE FUNCTION stablecoin.insert_transfer(start_ts timestamptz, end_ts timestamptz=now(), start_block numeric=0, end_block numeric=9e18) RETURNS integer
+LANGUAGE plpgsql AS $function$
+DECLARE r integer;
+BEGIN
+WITH transfers AS (
+    SELECT  symbol,
+            value / 10^decimals AS amount,
+            "from",
+            "to",
+            name AS coin_name,
+            tr.contract_address,
+            value AS amount_raw,
+            decimals,
+            evt_block_time AS block_time,
+            evt_tx_hash AS tx_hash,
+            evt_index
+    FROM erc20."ERC20_evt_Transfer" tr
+    INNER JOIN erc20.stablecoins st ON tr.contract_address = st.contract_address
+    LEFT JOIN prices.usd p
+        ON p.minute = date_trunc('minute', tr.evt_block_time)
+        AND p.contract_address = tr.contract_address
+        AND p.minute >= start_ts
+        AND p.minute < end_ts
+),
```

Comment:
Is it possible to replace this join with simpler time filter for `start_ts` and `end_ts`? e.g. 

```sql
...
INNER JOIN erc20.stablecoins st ON tr.contract_address = st.contract_address
WHERE date_trunc('minute', tr.evt_block_time) >= start_ts
        AND date_trunc('minute', tr.evt_block_time) < end_ts
...
```

Reply:
Yeah definitely.